charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -602,6 +608,37 @@ agent:
         }
       }
     }
+  windowsDefaultConfig:
+    {
+      "logs": {
+        "metrics_collected": {
+          "kubernetes": {
+            "enhanced_container_insights": true
+          },
+          "application_signals": { }
+        }
+      },
+      "traces": {
+        "traces_collected": {
+          "application_signals": { }
+        }
+      }
+    }
+  prometheus:
+    config:
+    targetAllocator:
+      enabled: false
+      image:
+        repository: cloudwatch-agent-target-allocator
```

Comment:
I think this section should be under operator since the TA image needs to be given to operator

Reply:
Nevermind we do not need this due this line on this code: https://github.com/aws/amazon-cloudwatch-agent-operator/blob/a8771928399017e02f7507f3e22efe9d228d6180/internal/manifests/targetallocator/container.go#L18